Automattic/jetpack
# SEO Upsell: update link to open in help center

Fixes https://github.com/Automattic/dotcom-forge/issues/8775

This will open the SEO tools upsell support post in the help center instead of in a new tab.
The support post is not localised as it is, so if the user has a non-English locale, the support post will open in English as things are.
This PR doesn't change this because adding support of `localizeUrl` into Jetpack is non-trivial as it comes with a lot of Calypso dependencies and would bloat Jetpack.
I think we can follow up with a PR like https://github.com/Automattic/wp-calypso/pull/94492 to ensure the post URL is localised.

## Proposed changes:

* Updates the SEO upsell link to open help center with the SEO upsell post

## Example
<img width="480" alt="Screenshot 2024-09-11 at 16 42 05" src="https://github.com/user-attachments/assets/d9982543-1e94-4b9c-aa8a-d1218e45a076">


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Test on simple sandbox with `bin/jetpack-downloader test jetpack add/seo-link-help-center`
* Go to post editor for a free site and follow the following video to confirm SEO upsell link loads in help center

https://github.com/user-attachments/assets/a3f9cb41-314a-4482-9d09-acab5d838dce
